src/lib/lessons.ts

```diff
@@ -27,6 +27,7 @@ const lessonQuery = groq`
   'free_forever': isCommunityResource,
   'path': '/lessons/' + slug.current,
   'thumb_url': thumbnailUrl,
+  'icon_url': coalesce(softwareLibraries[0].library->image.url, 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1567198446/og-image-assets/eggo.svg'),
```

Comment:
🔥

one problem here is that we aren't ranking the libraries which we do on the egghead-rails side so if multiple libs are associated we need to make sure they are "manually ranked" so that a "react" lesson doesn't present as some side topic

Reply:
Interesting! I didn't realize ranking was factoring into this on the egghead-rails side. I'll give that some thought.